code/modules/mob/living/carbon/human/human_attackalien.dm

```diff
@@ -37,13 +37,10 @@
 				visible_message("<span class='danger'>[M] has tackled down [src]!</span>", \
 					"<span class='userdanger'>[M] has tackled down [src]!</span>")
 			else
-				if (randn <= 99)

```

Comment:
Cool. Remember to check what happens to hardsuit helmets that get facehugged. I share the concern raised earlier about them being damaged/destroyed, then trying to redeploy them. What happens? Is the helmet lost forever? Is it simply deactivated (I would personally pick this one)? Will it runtime?